mysystehis/main.tex

```diff
@@ -0,0 +1,1155 @@
+  
+
+ To create the virtual world with Unity, first, we'll need to create the terrain by the tools Unity provides for us. After that, we can create my own model and then just simply drag and drop the model into the scene, if not, there're several websites i can get free 3d model from like turbosquid or free3d. For the Lighting and the shadow, Unity also has build-in tool just like other engines, which can help me create the light and shadow easily, adjust it as i want. Furthermore, to make things look good, we can apply some textures to all the objects, prefabs. We can find the texture on google, choose the suitable one for a specific object, and then just simply drag and drop them on that object, or making our own texture. Finally, in order to move around or interact with the object, a camera and player controller should be added into the scene, however, Unity already has the FPScontroller prefab which can be found in the standard assets, we can use it directly without doing anything, but it's still possible to create the character controller from scratch with C\# code.
+ \section{\Large Objective} 
+ 	In this project, since it's about building an RPG game, which is also a type of virtual world, I'll only focus on how to make the player move around the world, is able to interact with other objects around him. In this game, there may be some enemies that get in the way, the player should eliminate them which hurt the player. 
+ \section{\Large Aim} 
+ 	The aim is not necessarily on destroying the enemies, but focus on completing 
+ 	the quest that villagers in the village gave the player. When finishing a quest, the player should return back to the village, talk to the villagers that you're finished. Beside that, the game should also have an online chat tab , which you can chat with other players who are connecting to the same network while playing game, they may help you finishing the quest faster. 
+
+ 
+ 
+
+ \titleformat{\paragraph}
+ {\normalfont\normalsize\bfseries}{\theparagraph}{1em}{}
+ \titlespacing*{\paragraph}
+ {0pt}{1.25ex plus 1ex minus .2ex}{1.5ex plus .2ex}
+ 
+ \vspace{0.35 cm}
+  \section{\Large Programs, Materials and Methods}
+  	\subsection{External programs used}
+  		\subsubsection{3DS Max}
+ 		 3DS Max, a program developed by Autodesk, previously called 3D Studio Max, is a 3D computer graphics program for making 3D animations, models , images...
+ 		 This program was used in this project in order to edit UV mapping of some models, correct the position of the texture. 
+ 		 \subsubsection{Photoshop}
+ 		 Photoshop is a photo editing and graphic design software. It is developed by Adobe Systems for MacOS and Windows. It's not like other graphic design or photo editing softwares, it can create normal map, height map, occulusion map from a single texture. With this, this software will help me with the technique bump mapping, so the game's texture will look more realistic. 
```

Comment:
It is different from other graphic design or photo editing software in that it can create normal map...